assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -136,7 +141,7 @@ jobs:
   # Preparer
   batch_inference_preparer: 
     type: command
-    component: azureml:batch_inference_preparer:0.0.3
+    component: azureml:batch_inference_preparer:0.0.4
```

Comment:
we can keep the version `0.0.3` for all of the batch components as we have `0.0.2` in `azureml` registry.

Reply:
all changed back